models/models.go

```diff
@@ -118,7 +119,48 @@ func (m *RouteManager) Get(instanceId string) (*Route, error) {
 	}
 }
 
-func (m *RouteManager) Update(r *Route) error {
+func (m *RouteManager) Update(instanceId, domain, origin string, path string, insecureOrigin bool) error {
```

Comment:
let's save this for another PR. options will need to be moved into its own package because this individual passing of args happens in `models` and `utils`